Libraries/LibWeb/HTML/HTMLLIElement.h

```diff
@@ -6,8 +6,12 @@
 
 #pragma once
 
+#include <AK/GenericShorthands.h>
 #include <LibWeb/ARIA/Roles.h>
+#include <LibWeb/DOM/TreeWalker.h>
```

Comment:
What are these two new includes used for?

Reply:
My LSP+clangd setup seems to have auto-added those. I’ll try removing them and see if it still builds

Reply:
OK, removed those two includes (they’re unneeded)

Reply:
fwiw this kind of auto-insertion is why we have --header-insertion=none as one of the default clangd args recommended in the ... VsCode docs I believe.

Reply:
> fwiw this kind of auto-insertion is why we have --header-insertion=none as one of the default clangd args recommended in the ... VsCode docs I believe.

Thanks — I thought I had that set already in my clangd config, but apparently I don't. I'll add it